backend/app.py

```diff
@@ -1965,5 +1965,112 @@ def determine_subscription_tiers(subscription):
     return tiers
 
 
+################################################
+# Financial Doc Ingestion
+################################################
+
+from financial_doc_processor import *
+from utils import *
+from sec_edgar_downloader import Downloader
+from app_config import FILING_TYPES, BASE_FOLDER
+
+@app.route('/api/FinancialDocIngestion', methods=['POST'])
```

Comment:
hey @namtran6701 could you change HTTP method for the endpoint to GET for better semantic accuracy. Additionally, update the endpoint name to /api/SECEdgar/financialdocuments to align with the established naming convention for endpoints.